content/apps/using-oauth2.md

```diff
@@ -0,0 +1,157 @@
+# Using OAuth 2.0 to connect an App
+
+The PIM uses OAuth 2.0 to manage the authorization of Apps.
+
+![App activation diagram](../img/apps/app-activation-sequence-diagram.png)
+
+
+## Credentials
+
+To begin, you must obtain valid OAuth 2.0 client credentials by registering your App on the
+[Akeneo Marketplace](https://marketplace.akeneo.com/node/add/extension).
+
+Then, when any user want to connect his PIM to your App, the authorization process will go through the following steps.
+
+::: info
+In the following examples, we will use those fictional urls:  
+`https://my-pim.cloud.akeneo.com` for the PIM  
+`https://my-app.example.com/oauth/activate` for your App Activation URL  
+`https://my-app.example.com/oauth/callback` for your App Callback URL
+::::
+
+## Activation URL
+
+Your App must expose an activation URL.  
+When the PIM user sees your App in his PIM marketplace, he can click on the "Connect" button
+and he will be redirected to this activation URL with the PIM URL in the query.
```

Comment:
```suggestion
Your App must expose an activation URL.

The Akeneo PIM user has access to an App marketplace in his PIM. When he connects your App from its PIM, he will be redirected to the activation URL you provided. You will receive in the query the URL of the PIM where he comes from.
```

Reply:
Akeneo PIM users have access to an App marketplace in their PIM. When they connect your App from their PIM, they are redirected to the activation URL you provided. The PIM URL they come from is in the query you receive.